### PR TITLE
IA topics listed in docs

### PR DESCRIPTION
**Pull request by talsraviv**
@moollaza either this or linking to duck.co/ia… pros and cons…

### CODE REVIEW

File: duckduckhack/instant-answer-display/display_reference.md [hunk 1]
```diff
@@ -77,6 +79,54 @@ The name that will be used for your Instant Answer's AnswerBar tab. The Instant
...
+- jobs
+- literature
+- local
+- math
+- movies
+- music
+- news
+- parking
+- politics
+- producthunt
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/display_reference.md]: this one is a bug, should be removed ;)

File: duckduckhack/instant-answer-display/display_reference.md [hunk 2]
```diff
@@ -44,7 +44,9 @@ The `id` should match the name of your callback function. For example, if your c
 
 ## `name` *string* [required]
 
-The name that will be used for your Instant Answer's AnswerBar tab. The Instant Answer Framework will determine the final tab name, but it's best to provide a category or topic that describes the kind of information your Instant Answer provides. Here are some examples:
+The name that will be used for your Instant Answer's AnswerBar tab. The Instant Answer Framework will determine the final tab name, but it's best to provide a category or topic that describes the kind of information your Instant Answer provides.
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/display_reference.md]: Can remove the part abut the framework determining the name -- whatever you set for the `name` prop is what will be used for the tab.

File: duckduckhack/instant-answer-display/display_reference.md [hunk 3]
```diff
@@ -77,6 +79,54 @@ The name that will be used for your Instant Answer's AnswerBar tab. The Instant
     </tbody>
 </table>
 
+Ideally your Instant Answer `name` should be one of the existing topics:
+
+- answer
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/display_reference.md]: Should mention that this one is probably safest bet if none of the others work.

The overarching goal is to not have **any** brand names as the tabs. I.E. Videos for YouTube, Gaming for Twitch, Products for Amazon, etc

File: duckduckhack/instant-answer-display/display_reference.md (hunk 1 above)
Comment (talsraviv) [inline duckduckhack/instant-answer-display/display_reference.md]: product hunt I assume?

**Comment by talsraviv** [comment]
@moollaza thanks for reminding me - let me know if I missed something.

**Comment by talsraviv** [comment]
@zekiel take a look

File: duckduckhack/instant-answer-display/display_reference.md [hunk 4]
```diff
@@ -44,7 +44,13 @@ The `id` should match the name of your callback function. For example, if your c
 
 ## `name` *string* [required]
 
-The name that will be used for your Instant Answer's AnswerBar tab. The Instant Answer Framework will determine the final tab name, but it's best to provide a category or topic that describes the kind of information your Instant Answer provides. Here are some examples:
+The name that will be used **in the tab** above the Instant Answer, in the AnswerBar. 
+
+**Tab names should ideally be one word nouns** (Images, Videos, Products, Audio, Answer, News, etc.) that describe the type of content being shown. We recommend not having any brand names as the tabs. Instead, it's best to use the general topic name. For example, 'videos' for YouTube, 'gaming' for Twitch, 'products' for Amazon, and so on.
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/display_reference.md]: Would reword as `"We will not accept any brand names for tabs"`. It's really more than a requirement, rather than recommendation.

File: duckduckhack/instant-answer-display/display_reference.md [hunk 5]
```diff
@@ -77,6 +83,53 @@ The name that will be used for your Instant Answer's AnswerBar tab. The Instant
...
+- Health 
+- Images 
+- Jobs
+- Literature
+- Local
+- Math
+- Movies
+- Music
+- News
+- Parking
```
Comment (moollaza) [inline duckduckhack/instant-answer-display/display_reference.md]: This one is a mistake, should be remove internally as well!

**Comment by moollaza** [comment]
@talsraviv overall LGTM :+1:  A few minor fixes needed.
